model/database/metrics.yaml

```diff
@@ -131,3 +131,36 @@ groups:
     attributes:
       - ref: db.client.connection.pool.name
         requirement_level: required
+
+  - id: metric.db.client.response.returned_rows
+    type: metric
+    metric_name: db.client.response.returned_rows
+    brief: "The actual number of records returned by the database operation."
+    instrument: histogram
+    unit: "{row}"
+    stability: experimental
+    extends: attributes.db.client.minimal
+    attributes:
+      - ref: db.collection.name
+        requirement_level:
+          conditionally_required: >
+            If readily available. The collection name MAY be parsed from the query text,
+            in which case it SHOULD be the first collection name in the query.
+      - ref: db.system
+        # TODO: Not adding to the minimal because of https://github.com/open-telemetry/build-tools/issues/192
+        requirement_level: required
+      - ref: network.peer.address
+        brief: Peer address of the database node where the operation was performed.
+        requirement_level:
+          recommended: If applicable for this database system.
+        note: >
+          Semantic conventions for individual database systems SHOULD document whether `network.peer.*` attributes are applicable.
+          Network peer address and port are useful when the application interacts with individual database nodes directly.
+
+          If a database operation involved multiple network calls (for example retries), the address of the last contacted node SHOULD be used.
+      - ref: network.peer.port
+        requirement_level:
+          recommended: If and only if `network.peer.address` is set.
+      - ref: db.namespace
+        requirement_level:
+          conditionally_required: If available.
```

Comment:
sorry if this was discussed earlier, is there a reason for the set of attributes on `db.client.response.returned_rows` not to match the set of attributes on `db.client.operation.duration`? e.g.

```suggestion
    extends: attributes.db.client.with_query_and_collection
    attributes:
      - ref: db.system
        # TODO: Not adding to the minimal because of https://github.com/open-telemetry/build-tools/issues/192
        requirement_level: required
      - ref: network.peer.address
        brief: Peer address of the database node where the operation was performed.
        requirement_level:
          recommended: If applicable for this database system.
        note: >
          Semantic conventions for individual database systems SHOULD document whether `network.peer.*` attributes are applicable.
          Network peer address and port are useful when the application interacts with individual database nodes directly.

          If a database operation involved multiple network calls (for example retries), the address of the last contacted node SHOULD be used.
      - ref: network.peer.port
        requirement_level:
          recommended: If and only if `network.peer.address` is set.
      - ref: db.namespace
        requirement_level:
          conditionally_required: If available.
      - ref: db.query.text
        requirement_level: opt_in
```

Reply:
I believe it was added recently,..it was following what `db.client.operation.duration` had.